tools/setup-envtest/main.go

```diff
@@ -274,6 +276,10 @@ Environment Variables:
 			Input:       os.Stdin,
 			PrintFormat: printFormat,
 		}.Do(env)
+	case "version":
```

Comment:
Just to double check. This will result in a `setup-envtest version` command right?

Reply:
This is correct. I need to update the commands to express this.